core/src/test/java/org/mule/processor/chain/DefaultMessageProcessorChainTestCase.java

```diff
@@ -156,7 +156,6 @@ public void testMPChainWithNullReturnAtEnd() throws MuleException, Exception
         DefaultMessageProcessorChainBuilder builder = new DefaultMessageProcessorChainBuilder();
         builder.chain(new AppendingMP("1"), new AppendingMP("2"), new AppendingMP("3"), new ReturnNullMP());
         assertNull(builder.build().process(getTestEventUsingFlow("0")));
-        assertThat(RequestContext.getEvent(), is(nullValue()));
```

Comment:
I changed some of these tests to assert for the latest event processed to be in RC after the process is executed. I think in this case I was not able to do that but let me check it.